src/main/java/org/mule/extension/db/api/logger/LoggerApiPackage.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.db.api.logger;
+
+/**
+ * This is a class used just to export the package.
+ *
+ * @since 1.0
+ */
+public class LoggerApiPackage {
```

Comment:
delete this class and just export `MuleMySqlLogger`

Reply:
If I add the `MuleMySqlLogger` in `@Export()` the MySql dependency is required to be able to load the extension or it will fail to load the `annotation` content, but there is not another way to export a package without specifying the class.

Reply:
fair point, but explain it in the jdoc

Reply:
ready 👍